imports/plugins/included/product-variant/components/productGridItems.js

```diff
@@ -2,14 +2,26 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 import { Link } from "react-router-dom";
 import { formatPriceString, i18next } from "/client/api";
-import TableCell from "@material-ui/core/TableCell";
-import TableRow from "@material-ui/core/TableRow";
-import Checkbox from "@material-ui/core/Checkbox";
-import IconButton from "@material-ui/core/IconButton";
+import { TableCell, TableRow, Checkbox, IconButton, withStyles } from "@material-ui/core";
 import PencilIcon from "mdi-material-ui/Pencil";
+import CircleIcon from "mdi-material-ui/CheckboxBlankCircle";
+
+const styles = (theme) => ({
+  isVisible: {
+    color: theme.palette.colors.forestGreen300,
+    fontSize: theme.typography.fontSize * 1.25,
+    marginRight: theme.spacing(1)
```

Comment:
This is fine, but if the unit is `1` it's not required.

We should probably make a decision on whether to always use a number, even `1`, or leave it blank if it's `1`.

Doesn't matter to me either way, and will be a quick find / replace whichever way we decide.

Reply:
Ah, good to know. Add it to your future Spacing docs :)